pino.js

```diff
@@ -34,24 +34,40 @@ var levels = {
   debug: 20,
   trace: 10
 }
+Object.freeze(levels)
 
 var nums = Object.keys(levels).reduce(function (o, k) {
   o[levels[k]] = k
   return o
 }, {})
+Object.freeze(nums)
```

Comment:
`Object.freeze` has a very bad performance reputation. Given that all those globals are done to speed things up, we cannot probably afford this.